src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -53,6 +61,11 @@ static bool SendQueryInSingleRowMode(PGconn *connection, char *query);
 static bool StoreQueryResult(MaterialState *routerState, PGconn *connection,
 							 TupleDesc tupleDescriptor, int64 *rows);
 static bool ConsumeQueryResult(PGconn *connection, int64 *rows);
+static void RegisterRouterExecutorXactCallbacks();
```

Comment:
Without void in the params, that's an old style function declaration (i.e. taking arbitrary parameters, without checking).